theme/scripts/productDescription.ts

```diff
@@ -0,0 +1,83 @@
+export default(function() {
+	const ProductDescription = {
+		SELECTORS: {
+			tab: "[data-product-description-tab]",
+      content: "[data-product-description-content]",
+      tabsWrapper: ".ProductDescription__tabs",
```

Comment:
We're generally trying to stick to using `data-attributes` for handling DOM in JS. Allows us to distinguish from presentational attributes like classes. The activeTab class is cool though, although you could also use a data-attribute if you wanted.

Reply:
Cool that makes sense!